### PR TITLE
Update mongochef.rb

### PR DESCRIPTION
**Pull request by kievechua**
update to 3.0.0

### CODE REVIEW

File: Casks/mongochef.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask :v1 => 'mongochef' do
-  version '2.0.7'
+  version '3.0.0'
   sha256 'bc699e2c9fd2062acc4fdad5b519b9a1c95e53991c5c594b95b88b33bf17c00c'
```
Comment (vitorgalvao) [inline Casks/mongochef.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

Comment (kievechua) [inline Casks/mongochef.rb]: Done, thanks for letting me know :D

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by kievechua** [comment]
@vitorgalvao Squashing done too :)

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 29c030a833a210cdf45b754b78033da049fff345 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**